Client/src/components/Layout/Desktop/index.js

```diff
@@ -1,34 +1,33 @@
-import React, { Component } from 'react';
+import React, { Component } from "react";
 import { BrowserRouter, Route, Switch, Redirect } from "react-router-dom";
```

Comment:
use single quotation
```suggestion
import { BrowserRouter, Route, Switch, Redirect } from 'react-router-dom';
```